docs/modules/operation/pages/deep-dive/provisioning/foreign-source.adoc

```diff
@@ -0,0 +1,54 @@
+
+[[foreign-source-definition]]
+= Foreign Source Definition
+
+The default foreign source definition serves as a template that defines the services to detect on (DNS, FTP, ICMP, and so on), the scan interval for discovery, and the policies to use when provisioning.
+
+Policies determine entity persistence and/or set attributes on the discovered entities that control {page-component-title} management behavior.
+Provisiond applies the existing default foreign source definition unless you choose to modify it.
+
+This section describes how to edit the default foreign source definition.
+
+== Edit detectors and set scan interval
+
+. In the web UI, click the gear icon in the top right.
+. In the *Provisioning* area, choose *Manage Provisioning Requisitions*.
+. Click *Edit Default FS*.
+
++
+image::provisioning/FS_def.png[]
+
++
+NOTE: The screen displays the list of service detectors and a tab to view and define policies.
+Provisiond scans the services in the order in which the detectors appear in the list.
+
+. Click the appropriate icon to edit, delete, or move a service detector.
+.. You can also add parameters to a detector, including retries, timeout, port, and so on, by clicking the Edit icon and choosing *Add Parameter*.
```

Comment:
```suggestion
** Click the *Edit* symbol to edit a service detector's parameters.
To add more parameters (retries, timeout, port, and so on), click *Add Parameter* and define its name and value.
** Click the *Move* symbol to move a service detector.
** Click the *Delete* symbol to delete a service detector.
** If you make a change that you want to undo, click *Reset to Default* at the top-right of the screen.
```

Reply:
Reading this, we probably need to add a warning that clicking Reset to Default removes all of your changes. (Will test and verify.) If this is the case, someone could lose all the configuration they have done for 17 detectors ... I wonder if there is a way they can back it up.